src/main/java/com/moddy/server/service/application/HairModelApplicationRetrieveService.java

```diff
@@ -2,22 +2,78 @@
 import com.moddy.server.domain.hair_model_application.HairModelApplication;
 import com.moddy.server.domain.hair_model_application.repository.HairModelApplicationJpaRepository;
+import com.moddy.server.domain.prefer_hair_style.PreferHairStyle;
+import com.moddy.server.domain.prefer_hair_style.repository.PreferHairStyleJpaRepository;
+import com.moddy.server.service.designer.DesignerRetrieveService;
+import com.moddy.server.service.model.ModelRetrieveService;
 import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Sort;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
+import java.util.List;
+import java.util.stream.Collectors;
+
 @Service
 @RequiredArgsConstructor
 @Transactional(readOnly = true)
 public class HairModelApplicationRetrieveService {
 
     private final HairModelApplicationJpaRepository hairModelApplicationJpaRepository;
+    private final DesignerRetrieveService designerRetrieveService;
+    private final ModelRetrieveService modelRetrieveService;
+    private final PreferHairStyleJpaRepository preferHairStyleJpaRepository;
 
-    public String getApplicationCaptureUrl(Long applicationId){
+    public String getApplicationCaptureUrl(final Long applicationId){
         HairModelApplication application = hairModelApplicationJpaRepository.findById(applicationId).orElseThrow(() -> new NotFoundException(ErrorCode.NOT_FOUND_APPLICATION_EXCEPTION));
         return application.getApplicationCaptureUrl();
     }
 
+    @Transactional
+    public DesignerMainResponse getDesignerMainInfo(final Long designerId, final int page, final int size) {
+
+        Page<HairModelApplication> applicationPage = findApplicationsByPaging(page, size);
+        long totalElements = applicationPage.getTotalElements();
+
+        List<HairModelApplicationResponse> applicationResponsesList = applicationPage.stream().map(application -> {
+            Long modelId = application.getModel().getId();
+
+            List<PreferHairStyle> preferHairStyle = preferHairStyleJpaRepository.findTop2ByHairModelApplicationId(application.getId());
+
+            List<String> top2hairStyles = preferHairStyle.stream().map(hairStyle -> {
+                return hairStyle.getHairStyle().getValue();
+            }).collect(Collectors.toList());
+
+            HairModelApplicationResponse applicationResponse = new HairModelApplicationResponse(
+                    application.getId(),
+                    modelRetrieveService.getApplicationModelInfo(modelId).name(),
+                    modelRetrieveService.getApplicationModelInfo(modelId).age(),
+                    application.getModelImgUrl(),
+                    modelRetrieveService.getApplicationModelInfo(modelId).gender(),
+                    top2hairStyles
+            );
```

Comment:
```suggestion
            ApplicationModelInfoDto dto = modelRetrieveService.getApplicationModelInfo(modelId);
            HairModelApplicationResponse applicationResponse = new HairModelApplicationResponse(
                    application.getId(),
                    dto.name(),
                    dto.age(),
                    application.getModelImgUrl(),
                    dto.gender(),
                    top2hairStyles
            );
```
p2
함수를 한번 호출할 때마다 함수 내부에서 if문을 검증합니다.
if문 검증은 한번만 하면 충분하기 때문에 이런 식으로 함수를 한번만 호출해서 하는 것이 좋을 것 같습니다.